Client/PaymentRequest/PaymentRequestBodyHandler.swift

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+struct PaymentRequestBodyHandler: Decodable {
+    let name: String
+    let supportedInstruments: String
+    let details: String
+}
```

Comment:
Can we get some documentation on this? Also why is it named `Handler`? What's it handling?

Reply:
https://github.com/brave/brave-ios/pull/2327/files/5213ec4c60d67230d4b7ca5c98b3c7dd533ed4e1#diff-4d28bfc3eb95a2b647e02f0a574cddb6R40

I can rename this to `PaymentRequestBodyParser` if that works better?

Reply:
I just mean, most people won't know to go check a JS file to see what an instrument in this case is 😅 Like what's the difference between the body and the details

Re: naming, Parser doesn't really work either. It's not parsing anything. Should just be named based on the data in it. Maybe just `PaymentRequestBody` or whatever explains what's in it.

Reply:
Is `details` here supposed to turn into `PaymentRequestDetailsHandler` ?